localization/ekf_localizer/src/ekf_localizer.cpp

```diff
@@ -680,7 +679,7 @@ void EKFLocalizer::publishEstimateResult()
   pub_pose_cov_->publish(pose_cov);
 
   geometry_msgs::msg::PoseWithCovarianceStamped pose_cov_no_yawbias = pose_cov;
```

Comment:
```suggestion
  geometry_msgs::msg::PoseWithCovarianceStamped biased_pose_cov = pose_cov;
```

Reply:
Resolved in https://github.com/autowarefoundation/autoware.universe/pull/1787/commits/71f7c37e517e7ae4938af323588142b30ef15c95